public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -225,6 +225,7 @@ class MetricsPanelCtrl extends PanelCtrl {
     });
 
     var metricsQuery = {
+      timezone: this.dashboard.timezone,
```

Comment:
think this should be this.dashboard.getTimezone(),  if dashboard timezone is set to default it is null, and the timezone will be read from user or org preferences